Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -2383,6 +2383,13 @@ ErrorOr<String> Node::name_or_description(NameOrDescription target, Document con
         if (is<HTML::HTMLInputElement>(*element)) {
             auto& input = (const_cast<HTML::HTMLInputElement&>(static_cast<HTML::HTMLInputElement const&>(*element)));
             // https://w3c.github.io/html-aam/#input-type-image-accessible-name-computation
```

Comment:
Isn't this the image spec link? Should probably be https://w3c.github.io/html-aam/#input-type-button-input-type-submit-and-input-type-reset-accessible-name-computation

Reply:
Oops! Thanks much — fixed now